extensions/amp-animation/0.1/amp-animation.js

```diff
@@ -195,17 +197,11 @@ export class AmpAnimation extends AMP.BaseElement {
     this.setVisible_(false);
   }
 
-  /** @override */
-  activate(invocation) {
-    return this.startAction_(invocation);
-  }
-
   /**
    * @param {?../../../src/service/action-impl.ActionInvocation=} opt_invocation
    * @return {?Promise}
-   * @private
    */
-  startAction_(opt_invocation) {
+  startAction(opt_invocation) {
```

Comment:
Why does this need to be public?

Reply:
no need. I just made it public as the test used to call the public method activate to invoke this action and now that doesn't exist and I didn't want the test to make calls to a private method. I don't see an issue with this being public either.

Reply:
There's a small impact in that Closure compiler won't rename public function names, which yields a larger binary size.

Reply:
right. i can make private then and mark @visibleForTesting to indicate that it's directly called in tests.